xbmc/pvr/timers/PVRTimerInfoTag.h

```diff
@@ -640,5 +658,10 @@ class CPVRTimerInfoTag final : public ISerializable
   mutable std::shared_ptr<CPVRChannel> m_channel;
 
   mutable bool m_bProbedEpgTag = false;
+
+  unsigned int m_iAgeRating = 0; /*!< @brief age rating of the EPG entry used to create this timer */
```

Comment:
`unsigned int m_iAgeRating{0};` => coding guidelines

BTW: Here you have the type as `unsigned int` already (see my other comments).